modules/persistent-storage-hostpath-pod.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-hostpath.adoc
+
+[id="persistent-storage-hostpath-pod_{context}"]
+= Mounting the hostPath share in a privileged Pod
+
+After the PersistentVolumeClaim has been created, it can be used inside by an application. The following example demonstrates mounting this share inside of a Pod.
+
+.Prerequisites
+* A PersistentVolumeClaim exists that is mapped to the underlying hostPath share.
```

Comment:
Should we link to docs on creating this pvc?

Reply:
/lgtm after we consider this

Reply:
The steps for creating this PVC are actually described in the previous module (modules/persistent-storage-hostpath-static-provisioning.adoc). But unfortunately, our toolchain prevents the use of module-to-module xrefs. And because each 4.x module is written to serve as standalone content so that it may be reused anywhere else in our docs, we also can't refer to "the previous step." So, I am leaving it as is.